### DISCUSSION TITLE
Error while running step : step streamline

### DISCUSSION THREAD

**Discussion by CSP119** [Xilinx/finn#1004] [Q&A]
Hello all ! I am fairly new to finn and I was trying to generate the accelerator for my ml model and while running build.build_dataflow_cfg(model_file, cfg_estimates) , I ran into this error.
![Screenshot from 2024-03-17 20-12-32](https://github.com/Xilinx/finn/assets/98253660/b5b2fe8a-56bf-484d-be9d-29b94a613a25)
Any help would be much appreciated!

**Comment by aidanrhind** [comment]
Facing the same issue for my resnet model

**Reply by CSP119** [reply]
Did you try any other model ?

**Reply by aidanrhind** [reply]
I've tried it for the older "manual" way and the newer FINN build_dataflow approach and both seem to fail when applying streamlining. This discussion post produced the same errors I was facing for the old FINN approach for resnet50 while following steps from the cnv_w1a1 example notebook, but no solution has been found yet. https://github.com/Xilinx/finn/discussions/679

**Comment by yossimeshulam** [comment]
I get this issue in the next step

![image](https://github.com/Xilinx/finn/assets/73947067/532ffb3a-07da-4768-abed-f69dd59673dd)

**Reply by fpjentzsch** [reply]
Hi, did you try this with the current FINN dev branch? 

edit: a similar error was reported here: https://github.com/Xilinx/finn/discussions/1013
